src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanel.java

```diff
@@ -40,7 +40,11 @@
       triggerOnChange();
     });
     form.onSelectPemFile(path -> {
-      configuration.setPemFile(path);
+      if (path == null) {
```

Comment:
The other callbacks don't have any conditionals... This null checks should be done inside setPemFile(path)